documentation-site/pages/blog/introducing-react-view/index.mdx

```diff
@@ -0,0 +1,272 @@
+  const params = useView({
+    componentName: 'Button',
+    props: {
+      children: {
+        value: 'Hello',
+        type: PropTypes.ReactNode,
+        description: 'Visible label.',
+      },
+      onClick: {
+        value: '() => alert("click")',
+        type: PropTypes.Function,
+        description: 'Function called when button is clicked.',
+      },
+      disabled: {
+        value: false,
+        type: PropTypes.Boolean,
+        description: 'Indicates that the button is disabled',
+      },
+    },
+    scope: {
+      Button,
+    },
+    imports: {
+      'baseui/button': {
+        named: ['Button'],
+      },
+    },
+  });
+
+  return (
+    <React.Fragment>
+      <Compiler
+        {...params.compilerProps}
+        minHeight={62}
+        placeholder={Placeholder}
+      />
+      <Error msg={params.errorProps.msg} isPopup />
+      <Knobs {...params.knobProps} />
+      <Editor {...params.editorProps} />
+      <Error {...params.errorProps} />
+      <ActionButtons {...params.actions} />
+    </React.Fragment>
+  );
+};
+```
+
+### Live Code Editing Only
+
+If you don't need the code generation or props documentation, you can also use React View as an editable source code with the live preview.
+
+<LiveEditor
+  initialCode={`() => {
+  const text: string = "Hey";
+  return <h3>{text}</h3>;
+}`}
+/>
+
+### Usage
+
+```tsx
+import {useView, Compiler, Editor, Error} from 'react-view';
+import presetTypescript from '@babel/preset-typescript';
+
+export default () => {
+  const params = useView({
+    initialCode: `() => {
+      const text: string = "Hey";
+      return <h3>{text}</h3>;
+    }`,
+    scope: {},
+    onUpdate: console.log,
+  });
+
+  return (
+    <React.Fragment>
+      <Compiler {...params.compilerProps} presets={[presetTypescript]} />
+      <Editor {...params.editorProps} language="tsx" />
+      <Error {...params.errorProps} />
+    </React.Fragment>
+  );
+};
+```
+
+This example also demonstrates **how to opt-in into TypeScript support**. Since **we use babel** as the compiler, you can simply add additional presets (`preset-react` is applied by default). If you are using additional components or dependencies, you need to pass them through `scope`.
+
+You could also pass just a naked JSX element:
+
+<LiveEditor initialCode={`<h3>Hey</h3>`} />
+
+or pretty much anything that could be executed after the return statement of JS function:
+
+<LiveEditor initialCode={`2 + 5`} />
+
+## So What Really Is React View?
+
+**It is a set of tools** that aspires to close the gap between users, developers and designers of component libraries. React View aims to make documentations more interactive and useful.
+
+## Future
+
+**This is just a start for us**. We always gather the feedback and keep moving forward. There is already a [list of things](https://github.com/uber/react-view/issues/2) we want to improve. And there is also a long-term vision about how far the automatic code generation could get us.
```

Comment:
should we also encourage folks to take those issues on? if we'd break it down to multiple issues, with the same label, we could still link to them and would make it easier for folks to grab only one

Reply:
Yea, this was just a quick laundry list. We can probably split it and add more details later after the publishing (assuming there is enough interest)?